configure.ac

```diff
@@ -484,7 +484,8 @@ AC_CHECK_DECL([mach_absolute_time],
 AC_CHECK_DECLS([__builtin_ctz])
 
 AC_CHECK_DECLS([ffs], [], [],
-  [[#include <strings.h>]])
+  [[#define _XOPEN_SOURCE
```

Comment:
This needs to have a value of at least `700` for glibc. The “default” features complicate the observation of this requirement, but for example, this does not compile:

```c
#define _POSIX_C_SOURCE 200809
#define _XOPEN_SOURCE
#include <strings.h>
int main() { return ffs(1); }
```

and this does:

```c
#define _POSIX_C_SOURCE 200809
#define _XOPEN_SOURCE 700
#include <strings.h>
int main() { return ffs(1); }
```

See also [Feature Test Macros](https://www.gnu.org/software/libc/manual/html_node/Feature-Test-Macros.html).

Reply:
Fixed.